samples/database/TableCreationQueries.txt

```diff
@@ -0,0 +1,9 @@
+-------------------Create Table Queries------------------------
```

Comment:
Is this file necessary?

Reply:
Added it to make things easy if we are creating a db script in future. (not necessary now since we are using an embedded db)

Reply:
Might as well make it a `.sql` file now itself.

Reply:
make this a '.sql' file instead